src/utils/dateRange.ts

```diff
@@ -25,6 +25,11 @@ export function getLocalizedMonth(year, month, abbreviate: boolean = false) {
   return monthName;
 }
 
+// returns localized month name and passed in date
+export function getLocalizedAbbrMonthCost(date, month) {
+  return getLocalizedMonth(date, month, true) + ' ' + date;
+}
+
```

Comment:
We shouldn't concatenate strings. It would be preferable to have a localized message like: `'{month} {date}'`.